app/helpers/i18n_short_hand_helper.rb

```diff
@@ -0,0 +1,194 @@
+module I18nShortHandHelper
```

Comment:
This should be a pull request on @huerlisi gem.
Fork it, add it to your fork, make a pull request for simon and include the i18n_rails_helper gem via github from your fork